test/resources/SimpleAudioPlayer.js

```diff
@@ -82,15 +96,24 @@ SimplePlayer.prototype.handle = function () {
         this.play(podcastFeed[podcastIndex], 0, "ENQUEUE", podcastIndex, lastIndex);
 
     } else if (requestType === "AudioPlayer.PlaybackStarted") {
-        // We simply respond with true to acknowledge the request
-        this.context.succeed(true);
+        // Dom something async to that we have waited for it to finish
+        const self = this;
```

Comment:
if you want to avoid this, by using arrow functions in the setTimeout the "this" scope is mantained.